Pod/Classes/XExtensionItem.h

```diff
@@ -96,10 +140,51 @@ typedef UIImage *(^XExtensionItemThumbnailProvidingBlock)(CGSize suggestedSize,
  */
 @property (nonatomic, copy) NSDictionary *userInfo;
 
+/**
+ Add entries from a dictionary-serializable custom object to this paramter object’s `userInfo` dictionary.
+ 
+ @param dictionarySerializable Object whose entries should be added to this paramter object’s `userInfo` dictionary.
+ */
+- (void)addEntriesToUserInfo:(id <XExtensionItemDictionarySerializing>)dictionarySerializable;
```

Comment:
That actually makes a good deal of sense. So perhaps @mattbischoff's proposed name change is the best solution? Or maybe we could make the protocol more specific (just spitballing, `XExtensionItemParameterAddition` or something) so that the method name can be a little more concrete?